rest/models/src/main/java/com/connexta/store/rest/models/MetadataInfo.java

```diff
@@ -0,0 +1,95 @@
+package com.connexta.store.rest.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.net.URL;
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/** Information representing Metadata. */
+public class MetadataInfo {
+
+  @JsonProperty("metadataType")
+  private String metadataType;
+
+  @JsonProperty("location")
+  private URL location;
+
+  public MetadataInfo metadataType(String metadataType) {
+    this.metadataType = metadataType;
+    return this;
+  }
+
+  @ApiModelProperty(example = "irm", required = true, value = "The type of metadata.")
+  @NotNull
+  @Size(min = 1, max = 80)
+  public String getMetadataType() {
```

Comment:
✏️ Should this have a regex too? It needs to be valid in a url.

Reply:
I'm not sure TBH. I'd say it's okay as string for now.

Reply:
👍 I just checked with Patrick. He said that Arch Team doesn't know what the regex should be yet, so we're good doing exactly what they did in the transform api with `@Size(min = 1, max = 80)`.